src/packages/shared-utils/package-actions/getAvailableActions.ts

```diff
@@ -11,7 +12,13 @@ export const getAvailableActions = (
   result: OsMainSourceItem
 ) => {
   const checks = PackageCheck(result);
-  return checks.planTypeIs([PlanType.MED_SPA])
+  const finalChecks = checks.planTypeIs([PlanType.MED_SPA])
     ? rules.filter((r) => r.check(checks, user)).map((r) => r.action)
     : [];
```

Comment:
This should remain _UNCHANGED_, as our eventual goal is for the function to simply return:
```typescript
rules.filter((r) => r.check(checks, user)).map((r) => r.action)
```

This is taking things in the opposite direction and breaking up where we control our Action auth logic. Check out the rules configs and figure out how your logic fits into that system.